exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -35,10 +35,10 @@
 /** Adapts OpenTelemetry objects to Jaeger objects. */
 @ThreadSafe
 final class Adapter {
-  private static final String KEY_LOG_MESSAGE = "message";
-  private static final String KEY_SPAN_KIND = "span.kind";
-  private static final String KEY_SPAN_STATUS_MESSAGE = "span.status.message";
-  private static final String KEY_SPAN_STATUS_CODE = "span.status.code";
+  protected static final String KEY_LOG_MESSAGE = "message";
```

Comment:
This should be package protected, also the class is final so in no way protected can be used.